src/App.js

```diff
@@ -33,4 +35,5 @@ export default App
 
 const AppWrapper = styled.div`
```

Comment:
No need to mention `App` again for me in the component name.
```suggestion
const Wrapper = styled.div`
```